hadoop-client/2.7.2.wso2v1/pom.xml

```diff
@@ -0,0 +1,227 @@
+                            javax.security.auth.*;version="[0.0.0,1.0.0)",
+                            javax.net.*;version="[0.0.0,1.0.0)",
+                            javax.naming.*;version="[0.0.0,1.0.0)",
+                            javax.management.*;version="[0.0.0,1.0.0)",
+                            javax.crypto.*;version="[0.0.0,1.0.0)",
+                            javax.annotation.*;version="[0.0.0,1.0.0)",
+                            io.netty.*;version="[4.0.0,5.0.0)",
+                            com.google.gson.*;version="[2.5.0,3.0.0)",
+                            org.apache.commons.io.*;version="[2.4.0,3.0.0)",
+                            com.google.protobuf;version="[2.5.0,3.0.0)",
+                            com.google.common.*;version="[19.0.0,20.0.0)",
+                            org.slf4j.*;version="[1.7.0,2.0.0)",
+                            javax.servlet.*;version="[2.6.0,3.0.0)",
+                            com.google.inject.*;version="[1.3.0,2.0.0)" ;resolution:=optional,
+                            com.jcraft.jsch.*;version="[0.0.0,1.0.0)" ;resolution:=optional,
+                            com.sun.javadoc.*;version="[0.0.0,1.0.0)" ;resolution:=optional,
+                            com.sun.jersey.* ;version="[0.0.0,1.0.0)" ;resolution:=optional,
+                            com.sun.jndi.ldap.*;version="[0.0.0,1.0.0)" ;resolution:=optional,
+                            com.sun.tools.doclets.standard.* ;version="[0.0.0,1.0.0)" ;resolution:=optional,
+                            jdiff.*;version="[0.0.0,1.0.0)" ;resolution:=optional,
+                            org.apache.avro.*;version="[0.0.0,1.0.0)" ;resolution:=optional,
+                            org.apache.commons.compress.archivers.tar.*;version="[1.4.0,2.0.0)" ;resolution:=optional,
+                            org.apache.commons.daemon.*;version="[1.0.0,2.0.0)" ;resolution:=optional,
+                            org.apache.commons.net.* ;version="[3.1.0,4.0.0)" ;resolution:=optional,
+                            org.apache.curator.* ;version="[2.7.0,3.0.0)" ;resolution:=optional,
+                            org.apache.directory.server.kerberos.shared.keytab.* ;version="[2.0.0,3.0.0)" ;resolution:=optional,
+                            org.codehaus.jackson.* ;version="[1.9.0,2.0.0)" ;resolution:=optional,
+                            org.fusesource.leveldbjni.*;version="[1.8.0,2.0.0)" ;resolution:=optional,
+                            org.iq80.leveldb.* ;version="[0.6.0,1.0.0)" ;resolution:=optional,
+                            org.mortbay.*;version="[6.1.0,7.0.0)" ;resolution:=optional,
+                            org.znerd.xmlenc.* ;version="[0.0.0,1.0.0)" ;resolution:=optional,
+                            sun.misc.* ;version="[0.0.0,1.0.0)" ;resolution:=optional,
+                            sun.net.*;version="[0.0.0,1.0.0)" ;resolution:=optional,
+                            sun.nio.ch.*;version="[0.0.0,1.0.0)" ;resolution:=optional
```

Comment:
shall we have all the ranges as property and have it here? Currently some have property and some don't.